src/nip47/client.ts

```
@@ -34,26 +34,37 @@ import { SecurityValidationError } from "../utils/security-validator";
+  // Extract pubkey from pathname (remove leading slash)
+  const pubkey = parsed.pathname.slice(1);
   if (!pubkey) {
     throw new Error("Missing pubkey in NWC URL");
   }
 
-  // Parse query parameters
-  const params = new URLSearchParams(queryString);
+  // Parse query parameters using the URL's searchParams
   const relays: string[] = [];
-  params.getAll("relay").forEach((relay) => relays.push(relay));
+  parsed.searchParams.getAll("relay").forEach((relay) => relays.push(relay));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)